plugins/queue/mmap/queue_operation.go

```diff
@@ -20,11 +20,14 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 // SOFTWARE.
 
+//go:build !windows
```

Comment:
What's the difference?

```
//go:build !windows
// +build !windows
```

Reply:
They are the same, I'm using the go@1.17 version in my local machine. So It auto-generates by `go fmt`, being deleted.